docs/docs/cmd/teams/channel/channel-set.md

```diff
@@ -10,11 +10,17 @@ m365 teams channel set [options]
 
 ## Options
 
-`-i, --teamId <teamId>`
-: The ID of the team where the channel to update is located
+`-i, --teamId [teamId]`
+: The ID of the team where the channel to update is located. Specify either `teamId` or `teamName` but not both
 
-`--name <name>`
-: The name of the channel to update
+`--teamName [teamName]`
+: The display name of the team where the channel to update is located. Specify either `teamId` or `teamName` but not both
+
+`-c, --id [id]`
```

Comment:
In accordance with the Spec, could you change the short `-c` to `-i` here?

Could you also do this in the `channel get` command?